types/highland/index.d.ts

```diff
@@ -783,7 +783,7 @@ declare namespace Highland {
 		 * // after 100ms => 1, 2, 3, 4
 		 * // after 200ms => 1, 2, 3, 4, 5
 		 */
-		ratelimit<U>(num: number, ms: number): Stream<U>;
+		ratelimit<R>(num: number, ms: number): Stream<R>;
```

Comment:
The `R` type is defined by the `Stream<R>` interface, so it shouldn't be shadowed by another `R` here at the method level. I mean that there should be no `<R>` after `ratelimit`. Have a look at `append` or `compact`, which like `ratelimit` don't change the type of the elements of the stream.

Reply:
Oh I totally missed that! Thanks for the tip, I'll change that now.